WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/AvatarTrainView.swift

```diff
@@ -47,6 +47,10 @@ final class AvatarTrainView: UIView {
         }
     }
 
+    override func layoutSubviews() {
+        avatarStackView.arrangedSubviews.forEach({ configureBorder(for: $0) })
```

Comment:
This is totally nitpicky. But I wondered why this uses `arrangedSubviews` but the same call in `traitCollectionDidChange` uses `subviews` (`avatarStackView.subviews.forEach { configureBorder(for: $0) }`). Changing it didn't seem to make a difference. Can they be the same? If so, maybe it can be moved to a method to ensure it's always done the same way?

Reply:
Definitely! This has been addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/18470/commits/4bf0b3f953f5afdfb81e7ec062db50399d89a762. Thanks!